formula_lists/pypi_automatic_resource_update_list.json

```diff
@@ -0,0 +1,15 @@
+{
+  "ansible": false,
+  "ansible@2.8": false,
+  "cloudformation-cli": {
+    "extra_packages": ["cloudformation-cli-go-plugin", "cloudformation-cli-java-plugin"]
+  },
+  "diffoscope": "diffoscope[cmdline]",
+  "ipython": {
+    "extra_packages": ["ipykernel"]
```

Comment:
`ipykernel` can technically be an "instead of" vs an "extra" since `ipykernel` depends on `ipython`, but its version numbering doesn't match that of `ipython`, so it can't  be specified in the same way as `diffocsope`. The end result is the same and it's such a small optimization that it's probably not worth adding extra logic in for.

Reply:
Yeah, I looked at that but specifying `ipykernal` as the package name instead of `ipython` doesn't totally work for a few reasons:
1. The version issue you mentioned
1. This would treat the formula as if it were an `ipykernal` formula meaning that it would add `ipython` as a resource and wouldn't actually include `ipykernal` as a resource.

Once all this stuff is merged I'll make sure to go through these formulae and update the comments describing the resource issues.

Thanks for the review! I'll get this merged once everything in https://github.com/Homebrew/brew/pull/9185 is resolved.